### ISSUE TITLE
Using the plugin with Capacitor Ionic

### ISSUE THREAD

**Issue by pencium** [alantom1423/cordova-plugin-unityar#3]
Did you mange to get this plugin working with Capacitor Ionic? I can see you asked on Stackoverflow but didn't get an answer.

https://stackoverflow.com/questions/69722275/unityframework-in-capacitor-ionic-project

**Comment by alantom1423** [comment]
No. But I was able to integrate using a Capacitor plugin  rather than using Cordova plugin inside a capacitor project. Faced some issues while running in iOS.
